test/Test/Language/Javascript/StatementParser.hs

```diff
@@ -23,8 +23,21 @@ testStatementParser = describe "Parse statements:" $ do
         testStmt "{{}}"         `shouldBe` "Right (JSAstStatement (JSStatementBlock [JSStatementBlock []]))"
         testStmt "{{{}}}"       `shouldBe` "Right (JSAstStatement (JSStatementBlock [JSStatementBlock [JSStatementBlock []]]))"
 
-    it "if" $
+    it "if" $ do
         testStmt "if (1) {}"    `shouldBe` "Right (JSAstStatement (JSIf (JSDecimal '1') (JSStatementBlock [])))"
+        -- fix: fix ambiguity with block and object literal on if block position
+        -- testStmt "if (1) {x}"    `shouldBe` "Right (JSAstStatement (JSIf (JSDecimal '1') (JSStatementBlock [JSIdentifier 'x'])))"
```

Comment:
@erikd I need some help with this. I still can't figure it out a way to disambiguiate block and object on a `if statement`. The minified version would print something like:

`if(a){{x}}...`

Reply:
The inner `{}` is the object.

Reply:
Probably, this happend since it will try ObjectLiteral first (?).

Reply:
I'm fine with minor sensible changes to the tests as needed.